gqcp/tests/Domain/DiscreteDomain_test.cpp

```diff
@@ -0,0 +1,119 @@
+// This file is part of GQCG-GQCP.
+//
+// Copyright (C) 2017-2020  the GQCG developers
+//
+// GQCG-GQCP is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// GQCG-GQCP is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with GQCG-GQCP.  If not, see <http://www.gnu.org/licenses/>.
+
+#define BOOST_TEST_MODULE "DiscreteDomain"
+
+#include <boost/test/unit_test.hpp>
+
+#include "Domain/DiscreteDomain.hpp"
+#include "Domain/SimpleDomain.hpp"
+
+
+/**
+ *  Test if the bitstring representation of the domain is returned correctly.
+ */
+BOOST_AUTO_TEST_CASE(bitstring_representation) {
+    // Construct a discrete domain "" of size 8 where the indices 0, 2, 4 and 6 are occupied.
+    // Note that the bitstring representation of the domain should be read from right to left.
+    const std::vector<size_t> element_indices = {0, 2, 4, 6};
+
+    GQCP::DiscreteDomain discrete_domain_from_indices(element_indices, 8);
+    BOOST_CHECK_EQUAL(discrete_domain_from_indices.asString(), std::string("10101010"));
+
+    // The same as above, but with passing an unsigned integer to the constructor.
+    // GQCP::DiscreteDomain discrete_domain_from_unsigned_int(85, 8);
+    // BOOST_CHECK_EQUAL(discrete_domain_from_unsigned_int.asString(), std::string("10101010"));
```

Comment:
This test is commented out, is that by choice or by accident?

Reply:
Definitely by accident. The test is re-enabled in https://github.com/GQCG/GQCP/pull/1040/commits/76e69474816843229760c541144bf15177a98857.